src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -779,6 +779,7 @@ override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitWhileUntilLoopSta
 override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitWithStatement(Microsoft.CodeAnalysis.Semantics.IWithStatement operation) -> void
 override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitYieldBreakStatement(Microsoft.CodeAnalysis.Semantics.IReturnStatement operation) -> void
 override Microsoft.CodeAnalysis.SyntaxNode.ToString() -> string
+static Microsoft.CodeAnalysis.CaseInsensitiveComparison.StartsWith(string value, string possibleStart) -> bool
```

Comment:
We have "EndsWith", so it is hardly an issue to add "StartsWith", so I will assume we are ok with this addition.